src/java.base/windows/classes/sun/nio/fs/WindowsFileSystem.java

```diff
@@ -103,21 +103,12 @@ public Iterable<Path> getRootDirectories() {
 
         // iterate over roots, ignoring those that the security manager denies
         ArrayList<Path> result = new ArrayList<>();
-        @SuppressWarnings("removal")
-        SecurityManager sm = System.getSecurityManager();
```

Comment:
"ignoring those that the security manager denies"  can be removed from the method description.